Uppsala/UppEt20.xml

```diff
@@ -0,0 +1,524 @@
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="UppEt20">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title xml:id="t1">
+                    Masḥafa qəddāse
+                </title>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                            This file is licensed under the Creative Commons
+                            Attribution-ShareAlike 4.0. 
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0356Upp"/>
+                        <collection>O Etiop.</collection>
+                        <idno>O Etiop. 20</idno>
+                        <altIdentifier><idno>Löfgren 17</idno></altIdentifier>
+                        <altIdentifier><idno>Löfgren V</idno></altIdentifier>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1ra" to="192vb"/>
+                            <title type="complete" ref="LIT1960Mashaf"/>
+                            
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="1ra" to="5va"/>
+                                <title type="complete" ref="LIT1960Mashaf#PrayerKidan"></title>
+                                
+                                <msItem xml:id="ms_i1.1.1">
+                                    <locus from="1ra" to="3ra"/>
+                                    <title type="complete" ref="LIT1960Mashaf#Kidan1"></title>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.1.2">
+                                    <locus from="3ra" to="4va"/>
+                                    <title type="complete">Kidān za-qatr</title>
+                                    <incipit>
+                                        <hi rend="rubric">ጸጋ፡</hi> ዘእግዚአብሔር፡ የሀሉ፡ 
+                                        ምስሌክሙ<hi rend="rubric" rendition="#partialRubric">፨</hi>
```

Comment:
we only use `#` for `target `to already specified elements. e.g. if somewhere in the record you produced a note explaining what you mean under partial rubrication, with an `xml:id="partialRubric"` then you can refer to it through `#partialRubric` which the app will interpret as  a link to `http://betamasaheft.eu/UppEt20#partialRubric`, bringing the reader to that `note`

there is no other use for `#` prefix

is the rendition partialRubric explained anywhere in the guidelines? I may have missed it

Reply:
I copied this encoding from another record, so I will simply take the `#` away

Reply:
I added the `tagsDecl` as above and reintroduced the `#`